rllib/algorithms/appo/tests/test_appo_learner.py

```diff
@@ -95,17 +93,9 @@ def test_appo_loss(self):
             algo_config.validate()
             algo_config.freeze()
 
-            learner_group_config = algo_config.get_learner_group_config(
-                SingleAgentRLModuleSpec(
-                    module_class=algo_config.rl_module_spec.module_class,
-                    observation_space=policy.observation_space,
-                    action_space=policy.action_space,
-                    model_config_dict=policy.config["model"],
-                    catalog_class=algo_config.rl_module_spec.catalog_class,
-                )
+            learner_group = algo_config.build_learner_group(
+                env=algo.workers.local_worker().env
             )
-            learner_group_config.num_learner_workers = 0
-            learner_group = learner_group_config.build()
```

Comment:
how are we setting the equivalent of `num_learner_workers = 0`?

Reply:
Now via `config.resources(num_learner_workers=0)` (which is the default anyways so it has no effect on this test here).